### ISSUE TITLE
@nomiclabs/hardhat-solhint does not work

### ISSUE THREAD

**Issue by sherpya** [NomicFoundation/hardhat#3724]
### Version of Hardhat

2.12.7

### What happened?

```
NomicLabsHardhatPluginError: An error occurred loading the solhint formatter stylish
    at getFormatter (.../node_modules/@nomiclabs/hardhat-solhint/src/index.ts:21:11
```

`Caused by: Error: Cannot find module 'eslint/lib/formatters/stylish'`

calling:

https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-solhint/src/index.ts#L12

### Minimal reproduction steps

- create hardhat project using typescript and default settings
- add hardhat-solhint and enable in config
- run npx hardhat check
- 

### Search terms

solhint hardhat-solhint

**Comment by yhuard** [comment]
I have the same issue. I'll try to push a fix later today

**Comment by Oni-giri** [comment]
@yhuard Please consider addressing this issue along: https://github.com/NomicFoundation/hardhat/issues/1665

**Comment by yhuard** [comment]
@Oni-giri I'll have a look on my free time 👌

**Comment by mhm13dev** [comment]
@yhuard Just to update you:
You fixed this issue in #3726 but when I installed `hardhat-solhint` I still get this error.
When I checked in node_modules, your fix is not included
![image](https://user-images.githubusercontent.com/28213822/230482042-7a0b7719-441d-442a-b419-5cde08f72ccc.png)

**Comment by mhm13dev** [comment]
Okay I will use the patch-package for now
